modules/creating-a-machine-pool-cli.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="creating_machine_pools_cli_{context}"]
+= Creating a machine pool using the ROSA CLI
+
+You can create additional machine pools for your {product-title} (ROSA) cluster by using the ROSA CLI (`rosa`).
+
+[IMPORTANT]
+====
+The compute node instance types, autoscaling options, and node counts that are available to you depend on your ROSA subscriptions, resource quotas and deployment scenario. For more information, contact your sales representative or Red Hat support.
```

Comment:
I don't think that this note applies to ROSA since it is on-demand and the customer is immediately charged for their use.  So their compute instances would not be limited by their subscription purchases.

Reply:
@0kashi thank you for your review and for highlighting this. I will remove this admonition.